dialogue-core/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     compile 'com.netflix.concurrency-limits:concurrency-limits-core'
     compile 'com.palantir.conjure.java.runtime:client-config'
     compile 'com.palantir.safe-logging:preconditions'
+    compile 'io.dropwizard.metrics:metrics-core'
```

Comment:
pulling in tritium isn't sufficient?

Reply:
fixed, thanks

Reply:
ahh, I'm assuming this is what triggered https://github.com/palantir/tritium/issues/274

Reply:
@schlosna yep!